### PR TITLE
Work branch

### PR DESCRIPTION
**Pull request by mqashoo77**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,14 +1,103 @@
...
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Sharp:opsz,wght,FILL,GRAD@48,400,0,0"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+  </head>
+  <body>
+    <div class="main" id="main">
+      <div class="titel">
```
Comment (MahmoudAbdAlKareem) [inline index.html]: title

File: script.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+const todo = document.getElementById("todo");
+const done = document.getElementById("done");
+clearBtn.addEventListener("click", clearSearch);
+add.addEventListener("click", addTask);
+taskSearch.addEventListener("keyup", filterTasks);
+
+let tasks = [];
+
+function getTaskFromLocalStorage() {
+  let reTasks = JSON.parse(localStorage.getItem("tasks"));
```
Comment (MahmoudAbdAlKareem) [inline script.js]:
```
 const tasks = JSON.parse(localStorage.getItem("tasks"));
 return tasks ?? [];
  
```

File: script.js [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+  if (reTasks == null) {
+    tasks = [];
+  } else {
+    tasks = reTasks;
+  }
+}
+
+function updateLocalStorage() {
+  let taskString = JSON.stringify(tasks);
```
Comment (MahmoudAbdAlKareem) [inline script.js]: `const tasks = JSON.stringify(tasks);`

File: script.js [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+getTaskFromLocalStorage();
+render();
+
+function clearSearch() {
+  taskSearch.value = "";
+  render();
+}
+
+function filterTasks() {
+  var listItem = document.querySelectorAll("li");
```
Comment (MahmoudAbdAlKareem) [inline script.js]: ❌❌❌❌ no var please

File: script.js [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+  let taskName = taskSearch.value.toLowerCase();
```
Comment (MahmoudAbdAlKareem) [inline script.js:33]: const

File: index.html (hunk 1 above)
Comment (mqashoo77) [reply index.html]: Done

File: script.js (hunk 3 above)
Comment (mqashoo77) [reply script.js]: can't assign the same value (tasks)
ok 
am making some changes

File: script.js (hunk 5 above)
Comment (mqashoo77) [reply script.js:33]: Done

**Pull request merged**